test/login_test.exs

```diff
@@ -97,6 +97,14 @@ defmodule LoginTest do
     assert {:ok, %Postgrex.Result{}} = P.query(pid, "SELECT 123", [])
   end
 
+  test "env var default db name" do
+    System.put_env("PGDATABASE", "postgrex_test")
+    opts = []
+    assert {:ok, pid} = P.start_link(opts)
+    assert {:ok, %Postgrex.Result{}} = P.query(pid, "SELECT 123", [])
+    System.delete_env("PGDATABASE")
```

Comment:
If the test fails we won't reset, can you copy the dance done with `try do..after.. end` and `PGPORT` later in this file?